internal/runtimex/runtime_test.go

```diff
@@ -0,0 +1,9 @@
+package runtimex
+
+import "testing"
+
+func BenchmarkFastrand(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		Fastrand()
+	}
+}
```

Comment:
We need to make sure the `Fastrand` is compiled correctly, add a simple test here would be better.

Like 

```
func TestFastrand(t *testing.T) {
	var isRandom bool
	for i := 0; i < 3; i++ {
		x := Fastrand()
		y := Fastrand()
		if x != y {
			isRandom = true
			break
		}
	}
	if !isRandom {
		t.Fatal("Invalid fastrand results")
	}
}

```